.github/workflows/continuous-build.yml

```diff
@@ -59,6 +52,38 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: us-east-1
 
+      - name: Download package signing GPG secret key
+        if: ${{ runner.os == 'Linux' && github.event_name == 'push' }}
+        run: |
+          aws secretsmanager get-secret-value --region us-east-1 --secret-id "XRayDaemonSecrets" | jq -r ".SecretString" > aws-xray-secret.gpg
+          md5sum aws-xray-secret.gpg
+
+      - name: Import signing GPG key
+        if: ${{ runner.os == 'Linux' && github.event_name == 'push' }}
+        run: |
+          gpg --import aws-xray-secret.gpg
```

Comment:
I think you probably have to retrieve the GPG public key using to your GH instance using this - `wget https://s3.us-east-2.amazonaws.com/aws-xray-assets.us-east-2/xray-daemon/aws-xray.gpg`?

Reply:
We only need public key for signing the rpm. And we can generate the public part of the key using `gpg --armor --export -a "aws-xray@amazon.com" > aws-xray-public.gpg` so we don't need to fetch from S3.